validation/src/shared_table/mod.rs

```diff
@@ -487,6 +522,20 @@ impl SharedTable {
 		signed_statement
 	}
 
+	/// Import the erasure-encoded chunk of data.
+	///
+	/// Only imports `i`-th erasure-encoded chunk of data where `i` is the `local_index`.
+	pub fn import_erasure_chunk(&self, chunk: ErasureChunk) {
```

Comment:
i'm not convinced that the entry point for importing erasure chunks should actually be through the shared table, since we will usually get erasure chunks _after_ the attestation session has ended (and the table dropped)